tests/integration/read/read_replace/step1/package.json

```diff
@@ -1,7 +1,5 @@
 {
     "name": "read_dbr",
-    "main": "bin/index.js",
-    "typings": "bin/index.d.ts",
     "scripts": {
         "build": "tsc"
```

Comment:
Could you clarify why we need these changes?

Reply:
Sorry, I wrote a comment to clarify when we needed these changes, but I never hit submit! lol You read my mind XD

https://github.com/pulumi/pulumi/pull/12759#discussion_r1179654521